dart/lib/src/sentry_options.dart

```diff
@@ -189,6 +190,25 @@ class SentryOptions {
   /// Whether to send personal identifiable information along with events
   bool sendDefaultPii = false;
 
+  /// Wether [SentryEvent] deduplication is enabled.
+  /// Can be further configured with [exceptionsToKeepForDeduplication].
+  /// Shoud be set to true if
+  /// [SentryHttpClient] is used to capture failed requests.
+  bool enableDeduplication = true;
+
+  int _exceptionsToKeepForDeduplication = 5;
+
+  /// Describes how many exceptions are kept to be checked for deduplication.
+  /// This should be a small positiv integer in order to keep deduplication
+  /// performant.
+  /// Is only in effect if [enableDeduplication] is set to true.
+  int get exceptionsToKeepForDeduplication => _exceptionsToKeepForDeduplication;
+
+  set exceptionsToKeepForDeduplication(int count) {
+    assert(count >= 0);
```

Comment:
if 0 should be `enableDeduplication=false`, so I'd remove `=`?

Reply:
Good catch